skel.metadata.xml

```diff
@@ -16,6 +16,7 @@
     <email>exampledev@gentoo.org</email>
     <description>Primary maintainer</description>
   </maintainer>
+<!--   The email and names of maintainer are given below. -->
```

Comment:
I don't think this really adds anything and it's not even above the first example.